src/Evolution/Systems/GrMhd/ValenciaDivClean/TransportVelocity.hpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "PointwiseFunctions/Hydro/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/TransportVelocity.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace grmhd::ValenciaDivClean::Tags {
+/// \brief Compute tag for the transport velocity.
+///
+/// Compute item for the transport velocity, using
+/// \f$v_t^i=\alpha v^i-\beta^i\f$, with
+/// $v^i$ being the spatial velocity, $\alpha$ the lapse, and
+/// $\beta^i$ the shift.
+template <typename DataType, size_t Dim, typename Fr = Frame::Inertial>
+struct TransportVelocityCompute
```

Comment:
Move to `Hydro/TransportVelocity.hpp`. This doesn't depend on the GRMHD variables, unlike the quadrupole which used `TildeD`